Sources/MapboxNavigation/Resources/Assets.xcassets/debug.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "debug.pdf",
```

Comment:
Nice icon. If we intend to keep it around long-term, please add it to [navigation-ui-resources](https://github.com/mapbox/navigation-ui-resources/) (assuming it’s in the public domain).